src/ngraph_utils.h

```diff
@@ -156,6 +156,18 @@ tf::Status TFDataTypeToNGraphElementType(tf::DataType tf_dt,
 tf::Status TFTensorShapeToNGraphShape(const tf::TensorShape& tf_shape,
                                       ngraph::Shape* ng_shape);
 
+// Returns a pointer to a set containing all TensorFlow dtypes supported by the
+// nGraph bridge. This set is a singleton object.
+const std::set<tf::DataType>* NGraphDTypes();
```

Comment:
Why not a `const &` as opposed to `*`?

Reply:
I had a reason, but I'm realizing now that it was over-paranoid. Will change this.